### ISSUE TITLE
FFXIV Positional Audio plugin out of date

### ISSUE THREAD

**Issue by BigBrawler** [mumble-voip/mumble#4045]

**Comment by Kissaki** [comment]
We depend on someone to update it.

@EmperorArthur was the original author. Are you willing to update?

Otherwise @BigBrawler do you want to learn how to either check for and provide us the memory addresses of positional data from the game, or update it yourself? The current documentation is at https://wiki.mumble.info/wiki/HackPositionalAudio

**Comment by EmperorArthur** [comment]
Yeah, sorry I couldn't keep it updated.  I just had too much on my plate, and even minor updates require finding the values again.  Plus, everyone I play with swapped to Discord :/

In theory, the player information struct which contains this data probably doesn't change it's format that often, so instead of the modern hardcoded method you could use heuristics to find the information.

Another option is, the [FFXIV Zoom Hack](https://github.com/jayotterbein/FFXIV-Zoom-Hack) is often updated with pointers to the camera struct, so you could use that to work back to at least some starting value.  

Oh, if you are going to maintain this, please NEVER write to these values.  Writing the player position value teleports the player to whatever is written!

**Comment by NotNite** [comment]
Heya, stopping by again - that positional audio plugin just got accepted into the main repository, and it's currently in the testing period. In a few days I'll move it so all users can download it.
